code/game/gamemodes/setupgame.dm

```diff
@@ -81,3 +82,65 @@
 				assignedToBlock=blocks_assigned[G.block]
 			assignedToBlock.Add(G.name)
 			blocks_assigned[G.block]=assignedToBlock
+=======
```

Comment:
Will have to check w/ will on this one, but incoming diffs should be good to replace